nmigen_boards/butterstick.py

```diff
@@ -0,0 +1,155 @@
+import os
+import subprocess
+import shutil
+
+from nmigen.build import *
+from nmigen.vendor.lattice_ecp5 import *
+from .resources import *
+
+
+__all__ = ["ButterStickPlatform"]
+
+
+class ButterStickPlatform(LatticeECP5Platform):
+    device      = "LFE5U-85F"
+    package     = "BG381"
+    speed       = "8"
+    default_clk = "clk30"
+    resources   = [
+        Resource("clk30", 0, Pins("B12", dir="i"),
+                 Clock(30e6), Attrs(IO_TYPE="LVCMOS33")),
+
+        # LED Anodes
+        *LEDResources(pins="C13 D12 U2 T3 D13 E13 C16", attrs=Attrs(IO_STANDARD="LVCMOS33")),
```

Comment:
Should `IO_STANDARD` be `IO_TYPE` here? The `IO_STANDARD` attribute isn't recognized:
```
Warning: IOBUF 'led_0__io' attribute 'IO_STANDARD' is not recognised (on line 11)
```